### ISSUE TITLE
in src/server/base/rss.inc.php the construct function was written with just one underscore

### ISSUE THREAD

**Issue by kdpenha** [purplepixie/freenats#25]
in **src/server/base/rss.inc.php** the construct function was written with just one underscore and this was breaking the code

Line 22
```
function _construct()
	{
		global $NATS;
		$this->NATS = &$NATS;
	}
```

**Comment by purplepixie** [comment]
Resolved through the PR from @kdpenha (thank you!), and released in 1.30.1a https://www.purplepixie.org/freenats/release.php?version=1.30.1a

**Comment by ruanleoterio** [comment]
Nice work @kdpenha
